pom.xml

```diff
@@ -102,6 +102,14 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>1.8</source>
```

Comment:
Reasons for 1.8? Code, compiled with 1.8 will not execute in 1.7 JVM, whereas 1.7 bytecode will happily work in 1.8 JVM.